modules/planning/math/curve1d/quintic_spiral_path.cc

```diff
@@ -168,6 +168,29 @@ double QuinticSpiralPath::DeriveTheta(const std::size_t param_index,
   return derivative;
 }
 
+double QuinticSpiralPath::DeriveKappaDerivative(
+    const std::size_t param_index, const double r) const {
+  double s = param_ * r;
+  double s2 = s * s;
+  double s3 = s2 * s;
+  double s4 = s2 * s2;
+
+  double derivative =
+      5.0 * coef_deriv_[5][param_index] * s4 +
+      4.0 * coef_deriv_[4][param_index] * s3 +
+      3.0 * coef_deriv_[3][param_index] * s2 +
+      2.0 * coef_deriv_[2][param_index] * s +
+      coef_deriv_[1][param_index];
+
+  if (param_index == DELTA_S) {
```

Comment:
Would it be better to change the static const like THETA0 ~ DELTA_S to math common under planning?